python/datasources/uhc.py

```diff
@@ -145,150 +140,208 @@ def upload_to_gcs(self, _, **attrs):
+        breakdown_df = breakdown_df.drop(columns=std_col.generate_column_name(determinant, 'estimated_total'))
+
+    breakdown_df = breakdown_df.drop(columns=std_col.POPULATION_COL)
+    return breakdown_df
+
+
+def get_average_determinate_value(matched_row, measure_name, df, state):
+    """Gets the average value of two determinents, ignores null values.
+
+       matched_row: row in the dataset that matches the measure and demographic we are looking for
+       measure_name: measure name that we want to average with
+       df: the dataframe containing all informaiton
+       state: string state name"""
+
+    pres_breakdown_value, mid_breakdown_value = np.nan, np.nan
 
-                            pres_breakdown_value, mid_breakdown_value = np.nan, np.nan
+    if len(matched_row) > 0:
+        pres_breakdown_value = matched_row['Value'].values[0]
 
-                            if len(matched_row) > 0:
-                                pres_breakdown_value = matched_row['Value'].values[0]
+    matched_row_midterm = df.loc[
+        (df[std_col.STATE_NAME_COL] == state) &
+        (df['Measure Name'] == measure_name)]
 
-                            matched_row_midterm = df.loc[
-                                (df['State Name'] == state) &
-                                (df['Measure Name'] == measure_name)]
+    if len(matched_row_midterm) > 0:
+        mid_breakdown_value = matched_row_midterm['Value'].values[0]
 
-                            if len(matched_row_midterm) > 0:
-                                mid_breakdown_value = matched_row_midterm['Value'].values[0]
+    average_value = np.nanmean(
+        [pres_breakdown_value, mid_breakdown_value])
 
-                            average_value = np.nanmean(
-                                [pres_breakdown_value, mid_breakdown_value])
+    return average_value * 1000
 
-                            output_row[std_col.VOTER_PARTICIPATION_PER_100K] = average_value * 1000
 
-                        # for other determinants besides VOTER
-                        elif len(matched_row) > 0:
-                            pct = matched_row['Value'].values[0]
-                            if pct:
-                                if determinant in PER100K_DETERMINANTS:
-                                    output_row[UHC_DETERMINANTS[determinant]
-                                               ] = matched_row['Value'].values[0]
-                                # convert from % to per 100k
-                                else:
-                                    output_row[UHC_DETERMINANTS[determinant]
-                                               ] = matched_row['Value'].values[0] * 1000
+def estimate_total(row, condition_name_per_100k):
+    """Returns an estimate of the total number of people with a given condition.
 
-                output.append(output_row)
+       sample_per_100k: a percentage of people in a demographic with a given condition, represented as a per 100k
```

Comment:
I'm not sure I follow this comment; do you need to update the arguments in the doc string?

Reply:
it should say `per_100k of people` good catch